lib/resources/r4/financial_transaction.yaml

```diff
@@ -0,0 +1,52 @@
+---
+name: Financial Transaction
+field_name_base_url: http://hl7.org/fhir/R4/basic.html
+fields:
+- name: financial-transaction-method
+  description: Payment method codes
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method
+  binding:
+    description: Payment method codes
+    terminology:
+    - display: v2 Payment Method Code
+      system: https://terminology.hl7.org/CodeSystem/v2-0570
+      info_link: https://www.hl7.org/fhir/v2/0570/index.html
+    - display: Millennium payment method codes
```

Comment:
I dont think we support proprietary codes

Reply:
Removed https://github.com/cerner/fhir.cerner.com/pull/297/commits/0e8234a13abfe664bfb577f71a8bbc9a6bb9af88#diff-5125c86cde87f8439cb9d1bb859194e9L13-L16

Reply:
+1